### PR TITLE
#52415 patch

### PR DESCRIPTION
**Pull request by Quintis1212**
Hello everyone ) This is patch for correct right apostrophe display from ticket https://core.trac.wordpress.org/ticket/52415

### CODE REVIEW

File: packages/components/src/form-token-field/suggestions-list.js [hunk 1]
```diff
@@ -105,9 +105,32 @@ class SuggestionsList extends Component {
...
+						  return str;
+						}
+					  
+						return decodeHTMLEntities;
+					  })();
+					  suggestion.label=decodeEntities(label);
 					return (
 						<li
-							id={ `components-form-token-suggestions-${ this.props.instanceId }-${ index }` }
+							id={ `components-form-token-suggestions78-${ this.props.instanceId }-${ index }` }
```
Comment (ajitbohra) [inline packages/components/src/form-token-field/suggestions-list.js:133]: looks like typo `suggestions78` in id ?

Comment (Quintis1212) [reply packages/components/src/form-token-field/suggestions-list.js:133]: Hello ) I forgot to remove 78 from class name when I searching a function )

Comment (Quintis1212) [reply packages/components/src/form-token-field/suggestions-list.js]: I have remove 78 from class name )

**Comment by Quintis1212** [comment]
Please , help me with tests that are failed , what does its mean :

` TypeError: Cannot create property 'label' on string 'sound'

      128 | 						return decodeHTMLEntities;
      129 | 					  })();
    > 130 | 					  suggestion.label=decodeEntities(label);
          | 					  ^
      131 | 					return (`

**Pull request closed without merging**